website/settings/local

```diff
@@ -0,0 +1 @@
+SEARCH_ENGINE = 'elastic'
```

Comment:
@sloria I don't actually have push rights so you'd need to pull it / merge from me again unless you want to just remove the line yourself.